src/components/common/Header.jsx

```diff
@@ -1,53 +1,55 @@
 import React from "react";
+import { emailWhiteIcon, hindfloIcon, phoneWhiteIcon } from "./image";
 
 const Header = () => {
+    let pathName = window.location.pathname;
```

Comment:
create a utility of isActivePath and use that across project wherever is required

Reply:
done